vos/vos/commands/vcp.py

```diff
@@ -355,25 +382,40 @@ def copy(source_name, destination_name, exclude=None, include=None,
                     if ra_dec_match is not None:
                         cutout = ra_dec_match.group('cutout')
                 logging.debug("cutout: {}".format(cutout))
+                logging.debug('GLOBbing {}'.format(source_pattern))
                 sources = lglob(source_pattern)
                 if cutout is not None:
                     # stick back on the cutout pattern if there was one.
                     sources = [s + cutout for s in sources]
+
+                logging.debug('Sources are {}'.format(sources))
             for source in sources:
-                if source[0:4] != "vos:":
+                logging.debug('Checking source {}'.format(source))
+                if not _is_vos(source) and not vos.is_remote_file(source):
+                    logging.debug('Correcting path.')
                     source = os.path.abspath(source)
                 # the source must exist, of course...
                 if not access(source, os.R_OK):
                     raise Exception("Can't access source: %s " % source)
 
+                logging.debug('Source {} can be accessed.'.format(source))
+
                 if not args.follow_links and islink(source):
                     logging.info("{}: Skipping (symbolic link)".format(source))
                     continue
 
                 # copying inside VOSpace not yet implemented
-                if source[0:4] == 'vos:' and dest[0:4] == 'vos:':
+                if (_is_vos(source) or vos.is_remote_file(source))\
+                        and (_is_vos(dest) or vos.is_remote_file(dest)):
```

Comment:
I find this `_is_vos`, `_is_remote_file`, `is_remote` confusing. There should be only one remote in the client. For vos client remote means `vos` in scheme. For inventory storage means scheme not `file:`. 
This check will be to see if both source and destination are remote.